[MODULES] Fix CMakeLists.txt for wallpapers module.

SavanDev opened the pull request:
## Purpose

Add all wallpapers when the module is included (Not only one).

Adding and modifying in each folder the CMakeLists.txt to link the wallpapers with the build, and renaming to remove the blanks to be able to link them.

## Proposed changes

- Add all wallpapers to the ISO (If the module is included)
- Able to see how enable the module in ReadMe.txt

MailYouLater commented:
I like the changes, but I feel like they should be broken up between at least 2 commits, as some of these changes aren't really all that closely related, and some aren't described in the commit message. ... Some aren't even described in the pull request description.

SavanDev commented:
There I expanded the purpose a bit explaining what I changed, but I only modified the name of 2 files (by the blanks they had in it) and moved another 2 to the folder that contained the same wallpaper style.
The rest was to modify the existing CMakeLists to include the wallpapers and subfolders and I added the file in the other folders for the same function, and in the readme explain how to integrate the module.

Maybe I should have separated it into two or more commits to better detail each step, sorry.

SavanDev commented:
I done with the commits but if I make a mistake in something, say me.
